.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: go
+go:
+- 1.4.3
```

Comment:
I copied the file from lantern repo and tweaked it a bit, good to change it to newest version.